includes/orders/functions/orders.php

```diff
@@ -1282,3 +1293,62 @@ function edd_clone_order( $order_id = 0, $clone_relationships = false, $args = a
 
 	return $new_order_id;
 }
+
+/**
+ * Get the order status array keys that can be used to run reporting related to gross reporting.
+ *
+ * @since 3.0
+ *
+ * @return array An array of order status array keys that can be related to gross reporting.
+ */
+function edd_gross_order_statuses() {
```

Comment:
Should this be `edd_get_gross_order_statuses()`?

(I know this existed already in the 3.0 branch. Maybe bring up renaming this in a different PR?)

Reply:
Well, it's a brand new function and so far this PR is the only place it's used that I know of. It's moved in this PR because I was trying to use it and discovered it was in a namespaced file. I would be open to changing both of these to include `get` if @ashleyfae approves?